auth.default.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
It seems unlikely that that these json files will be used outside the scope of `scripts/swagger_auth.py`,

Please consider in-lining this file and `auth.secure_all.json` into `scripts/swagger_auth.py`.

Reply:
The idea is to allow customization on what endpoints to secure. Are you suggesting we don't allow users this level of customization?

Reply:
We should talk about this on Monday!